plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/outputspanel/console/DefaultOutputConsole.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.extension.machine.client.outputspanel.console;
+
+import com.google.gwt.core.client.Scheduler;
+import com.google.gwt.user.client.ui.AcceptsOneWidget;
+import com.google.inject.Inject;
+import com.google.inject.assistedinject.Assisted;
+
+import org.eclipse.che.ide.extension.machine.client.command.CommandConfiguration;
+
+/**
+ * Console panel for some text outputs.
+ *
+ * @author Valeriy Svydenko
+ */
+public class DefaultOutputConsole implements OutputConsole {
+
+    private final OutputConsoleView view;
+    private       String            title;
+
+    @Inject
+    public DefaultOutputConsole(OutputConsoleView view, @Assisted String title) {
+        this.view = view;
+        this.title = title;
+    }
+
+    /**
+     * Print message in the console.
+     *
+     * @param text
+     *         message which should be printed
+     */
+    public void printText(String text) {
+        view.print(text, false);
+        Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {
```

Comment:
It would be nice if you left here a short explanation why you're using deferred command